login_server/loginserver.py

```diff
@@ -326,24 +326,22 @@ def handle_client_message(self, msg):
 
     def handle_http_request_message(self, msg):
         if msg.env['PATH_INFO'] == '/status':
+            self.logger.info('Served status request via HTTP to peer ' + str(msg.peer))
             msg.peer.send_response(json.dumps({
                 'online_players': len(self.players),
                 'online_servers': len(self.game_servers)
             }, sort_keys=True, indent=4))
         elif msg.env['PATH_INFO'] == '/detailed_status':
+            self.logger.info('Served detailed status request via HTTP to peer ' + str(msg.peer))
```

Comment:
I'm not sure how to log HTTP requests other than this. Is there a more accurate way to identify where the request came from?

Reply:
I don't remember what peer prints exactly, but I think it includes the IP. If it does that I think it's enough.

Reply:
"msg.peer" logs this: `<login_server.httphandler.HttpHandler object at 0x000001B9F532DCA0>`
And I don't think that long hex value changes per request. For example, these requests were made from different origins but show the same log:
![image](https://user-images.githubusercontent.com/22832983/116165728-97be8580-a750-11eb-929e-e86abf211420.png)

I'm not sure what this means, and I don't know how to find what other data is included in "msg", sorry.

Reply:
Although, that hex value is different after the login server is restarted.

Reply:
Never mind, I've figured out how to log the request origin correctly now. See commit 7653061 below.